main.tf

```
@@ -100,3 +100,11 @@ resource "aws_vpc_ipv6_cidr_block_association" "default" {
     }
   }
 }
+
+resource "aws_default_route_table" "default" {
+  count                  = local.enabled ? 1 : 0
+  default_route_table_id = aws_vpc.default.id
+  tags = merge(module.label.tags, {
```

cc: @aknysh @Nuru what do you folks think about this current tag/name structure? Would it be better to do this? Thoughts?

```suggestion
  tags                   = module.label.tags
```

Or have an additional input

```suggestion
  tags                   = merge(module.label.tags, var.tags_default_route)
```

Where tags_default_tags contains default=true.